src/jobs/recommendation.service.ts

```
@@ -0,0 +1,625 @@
+        topJobTypes: [],
+        recommendationsByScore: [],
+      };
+    }
+
+    const totalRecommendations = recommendations.length;
+    const averageScore = recommendations.reduce((sum, rec) => sum + rec.score, 0) / totalRecommendations;
+    
+    const viewedRecommendations = recommendations.filter(rec => rec.isViewed);
+    const appliedRecommendations = recommendations.filter(rec => rec.isApplied);
+    
+    const clickThroughRate = viewedRecommendations.length / totalRecommendations;
```

ESLint

[error] Insert `⏎·····`

(prettier/prettier)